service/src/main/java/uk/gov/hmcts/reform/fpl/validation/groups/ValidateCaseNumberGroup.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.fpl.validation.groups;
+
+// JSR Validation interface used to validate the C21 Group
```

Comment:
Remove comment and didn't we name this to ValidateFamilyManCaseNumber ??

Reply:
We did, this interface accidentally got added back when I was fixing merge conflict! thanks, will delete